api/include/mfxdefs.h

```diff
@@ -45,6 +45,11 @@
 #endif
 
 
+#if defined(__clang__)
+  #pragma clang diagnostic ignored "-Wunknown-pragmas"
```

Comment:
In a public header?! No, no, no. That's a bad idea. Please, remove this change from the PR. Let's have a second round of warning fixes in another PR and deal with it there.